src/Fabulous.MauiControls/Program.fs

```diff
@@ -23,6 +23,9 @@ module MauiViewHelpers =
             | None -> ValueNone
             | Some attr -> ValueSome attr.Value
 
+    let defaultSyncAction (action: unit -> unit) =
```

Comment:
It would be great to call this function also in Program.fs

Reply:
Done